### PR TITLE
[JA] FAQ translation update

### PR DESCRIPTION
**Pull request by 0xhotchocolate**
Added missing Japanese translating on FAQ page. 
Since this is my first PR here, I keep the change small. As soon as I get to know the process better, will work on more!

### CODE REVIEW

File: content/faq.ja.md [hunk 1]
```diff
@@ -13,6 +13,33 @@ name: よくある質問
 ---
 # よくある質問
 
+<!--#{ using h4s for questions to keep them out of the right side nav (too cluttered when they display) and to provide appropriate text size for questions. #}-->
+#### XRP Ledger(XRPL)はRippleが所有するプライベートブロックチェーンですか？
+
+いいえ。XRPLは分散型のパブリックブロックチェーンです。トランザクション処理やコンセンサスに影響を与えるような変更は、ネットワークバリデーターの80％以上の承認が必要です。Rippleはネットワークへの貢献者の一員ですが、その権利は他の貢献者たちと同じです。ネットワークには150以上のバリデーターが参加しており、うちデフォルトのユニークノードリスト(UNLs)に登録されているものは35以上です。Rippleはこれらのうち6つのノードを運営しています。
```
Comment (mDuo13) [inline content/faq.ja.md]: Actually, this is a case where the English version is outdated: Of the 35 validators on the default UNL, Ripple is down to only 2 of them (not 6 anymore).

**Review by mDuo13 — APPROVED** [review]
This looks good. Thank you for the updates! I noticed one place where the translation isn't wrong but the text it was translating from is outdated; I'll leave this open for a little bit in case you want to update that. Otherwise, this should be OK to merge.

File: content/faq.ja.md (hunk 1 above)
Comment (0xhotchocolate) [reply content/faq.ja.md]: Hi @mDuo13 
Thanks for the comment. I opened #1665 to change the number on English doc. Once it's approved, I'll update this PR too!

File: content/faq.ja.md [hunk 2]
```diff
@@ -13,6 +13,33 @@ name: よくある質問
...
+
+
+#### XRPLではXRP以外の通貨も取引できますか？
+
+はい。XRPLは米ドル、ユーロ、石油、金、リワードポイントなど、任意の資産をトークン化できるように構築されており、どんな通貨でもXRPL上で発行することができます。成長中のXRPLコミュニティは様々なフィアットトークンや暗号通貨をサポートしており、これを裏付けています。
+
+
+#### XRPLは決済用だけではないのですか？
+
+いいえ。当初は決済のユースケース向けに開発されましたが、台帳であるXRP Ledgerとそのネイティブ・デジタルアセットであるXRPの両方は、さまざまな革新的ブロックチェーンのユースケースで更に人気を集めています。NFT発行のための新しい規約提案、XRPL Labsのスマートコントラクト機能に関するフック修正案、サイドチェーンの開発などが現在進行中です。
```
Comment (tequdev) [inline content/faq.ja.md]: This statement is out of date since NFT is now enabled on the mainnet.

It would need to be corrected in the English version and would be fine in a separate PR.

**Review by tequdev — APPROVED** [review]
As a Japanese expression, I think there is no point of inappropriateness.

File: content/faq.ja.md (hunk 2 above)
Comment (0xhotchocolate) [reply content/faq.ja.md]: updated English version on #1669

**Comment by 0xhotchocolate** [comment]
Applied the changes about Ripple owned nodes and XRPL use cases to Japanese version. 
This PR is ready to merge. Thanks for supporting my first PR @mDuo13 and @develoQ !!
